tests/clar.c

```diff
@@ -419,37 +423,33 @@ clar_test_run(void)
 {
 	if (_clar.argc > 1)
 		clar_parse_args(_clar.argc, _clar.argv);
-
 	if (!_clar.suites_ran) {
 		size_t i;
 		for (i = 0; i < _clar_suite_count; ++i)
 			clar_run_suite(&_clar_suites[i], NULL);
 	}
-
```

Comment:
Please avoid unnecessary code churn at places that aren't really touched by your patch.